src/add&remove.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable comma-dangle */
+let tasks = [];
+
+// const tasks = JSON.parse(localStorage.getItem('arrayOfTasks')) || [];
+
+function Task(description, taskIndex) {
+  this.description = description;
+  this.completed = false;
+  this.index = taskIndex;
+}
+export const createTask = () => {
+  getFromLocalStorage();
+  const taskDescription = document.getElementById('input-description');
+  const descriptionValue = taskDescription.value;
+  const taskIndex = tasks.length + 1;
+  const task = new Task(descriptionValue, taskIndex);
+  tasks.push(task);
+  saveToLocaleStorage();
+  tasks = [];
+  taskDescription.value = '';
+};
+
+const saveToLocaleStorage = () => {
+  localStorage.setItem('arrayOfTasks', JSON.stringify(tasks));
+};
+// saveToLocaleStorage();
+
+const getFromLocalStorage = () => {
+  if (localStorage.getItem('arrayOfTasks')) {
+    const newTasks = JSON.parse(localStorage.getItem('arrayOfTasks'));
+    for (let i = 0; i < newTasks.length; i++) {
+      tasks.push(newTasks[i]);
+    }
+  }
+};
+
+export const editHandler = (event) => {
+  tasks = [];
+  getFromLocalStorage();
+  localStorage.setItem('arrayOfTasks', JSON.stringify([]));
+  console.log('tasks array', tasks);
+
+  const foundTask = tasks.find(
+    (task) => task.description === event.target.value
+  );
+  console.log('from remove func', foundTask);
+  event.target.addEventListener('blur', () => {
+    tasks.forEach((task) => {
+      if (foundTask.index === task.index) {
+        task.description = event.target.value;
+      }
+    });
+    tasks = tasks.filter((task) => {
+      if (task.description.replace(/\s/g, '') === '') {
+        const { parentElement } = event.target;
+        parentElement.remove();
+        return false;
+      }
+      return true;
+    });
+    localStorage.setItem('arrayOfTasks', JSON.stringify(tasks));
+  });
+};
+
+// const removeHandler = (event) => {
+//   tasks = [];
+//   getFromLocalStorage();
+//   localStorage.setItem('arrayOfTasks', JSON.stringify([]));
+//   console.log('tasks array', tasks);
+// };
```

Comment:
- [ ] Removing commented code not only makes it harder to find later, but it also makes it so people in the future don't know it existed before. For finding the code, there are [tools and git commands](http://stackoverflow.com/q/278192/971592) to help you look at the history of a file. Please remove all commented code and unnecessary files from your code base :smile: